src/pages/languages.js

```diff
@@ -0,0 +1,159 @@
+        break;
+      case 2:
+        reduced.complete.push(language);
+        break;
+    }
+    return reduced;
+  },
+  {complete: [], incomplete: [], partial: []},
+);
+
+type Props = {
+  location: Location,
+};
+
+const Languages = ({location}: Props) => (
+  <Layout location={location}>
+    <Container>
+      <div css={sharedStyles.articleLayout.container}>
+        <div css={sharedStyles.articleLayout.content}>
+          <Header>Languages</Header>
+          <TitleAndMetaTags title="React - Languages" />
+
+          <div css={sharedStyles.markdown}>
+            <p>
+              The React documentation is available in the following languages:
+            </p>
+
+            <LanguagesGrid languages={complete} />
+
+            <h2>Partially Translated</h2>
+            <LanguagesGrid languages={partial} />
+
+            <h2>Needs Contributors</h2>
+            <LanguagesGrid languages={incomplete} />
+
+            <p>
+              Don't see your language above?{' '}
+              <a
+                href="https://github.com/reactjs/reactjs.org-translation#reactjsorg-translation"
+                target="_blank"
+                rel="noopener">
+                Let us know
+              </a>
+              .
+            </p>
+          </div>
+        </div>
+      </div>
+    </Container>
+  </Layout>
+);
+
+const LanguagesGrid = ({languages}) => (
+  <ul
+    css={{
+      display: 'flex',
+      flexWrap: 'wrap',
+      marginLeft: -20,
+    }}>
+    {languages
+      .sort((a, b) => a.code.localeCompare(b.code))
+      .map(({code, name, status, translated_name}) => (
+        <Language
+          key={code}
+          code={code}
+          name={name}
+          status={status}
+          translatedName={translated_name}
+        />
+      ))}
+  </ul>
+);
+
+const Language = ({code, name, status, translatedName}) => {
+  const prefix = code === 'en' ? '' : `${code}.`;
+
+  return (
+    <li
+      css={{
+        paddingLeft: 20,
+        paddingTop: 20,
+        borderTop: '1px dotted #ececec',
+        paddingBottom: 20,
+        width: '100%',
+        listStyle: 'none',
+
+        [media.size('small')]: {
+          width: '50%',
+        },
+
+        [media.size('medium')]: {
+          width: '33.33%',
+        },
+
+        [media.greaterThan('large')]: {
+          width: '25%',
+        },
+      }}
+      key={code}>
+      <div css={{}}>{name}</div>
+      <div
+        css={{
+          fontSize: 22,
+          fontWeight: 'bold',
+          marginBottom: 8,
+          marginTop: 8,
+        }}>
+        {status === 0 && translatedName}
+        {status > 0 && (
+          <a href={`https://${prefix}reactjs.org/`} rel="nofollow">
+            {translatedName}
+          </a>
+        )}
+      </div>
+      <div css={{marginTop: 10}}>
+        <a
+          css={{
+            fontSize: 12,
+          }}
+          href={`https://github.com/reactjs/${prefix}reactjs.org/`}
```

Comment:
Maybe it's better to link to the [progress issue](https://twitter.com/) for each language since it has the actual instructions for contributing.

It's going to be issue #1 for all languages (since it's auto-generated) except for Spanish, Japanese, and Simplified Chinese, for which it's #4 by a lucky coincidence.

Reply:
I think linking to the main repo makes more sense. The progress issue seems more transient. I think the repos should rely on something more standard (like the readme or `CONTRIBUTING.md`) for guidelines rather than a specific issue.

I think i'm going to leave this as-is.